tools/wake-format/main.cpp

```diff
@@ -198,7 +198,7 @@ int main(int argc, char **argv) {
   FILE *user_warn = stdout;
   user_warn = fopen("/dev/null", "w");
   if (auto_find_files) {
-    bool ok;
+    bool ok = true;
     wakefiles = find_all_wakefiles(ok, true, false, ".", ".", user_warn);
```

Comment:
Can you set it to true inside the function as well? I don't like functions with such difficult and hidden pre-reqs. We should set it true in both places so that neither side of the API is making an assumption.